--- rpython/translator/goal/translate_org.py	2018-05-22 09:32:08.000000000 +0200
+++ rpython/translator/goal/translate.py	2018-05-22 09:54:20.000000000 +0200
@@ -314,8 +314,14 @@
             except EnvironmentError:
                 pass
 
+        custom_tasks = os.environ.get('PYPY_CUSTOM_TASKS')
+
         try:
-            drv.proceed(goals)
+            # import pdb;pdb.set_trace()
+            if custom_tasks:
+                [drv.tasks[task][0]() for task in custom_tasks.split(',')]
+            else:
+                drv.proceed(goals)
         finally:
             drv.timer.pprint()
     except SystemExit:
