--- pypy3/pypy/goal/targetpypystandalone.py	2018-04-24 08:05:26.000000000 +0200
+++ pypy3/pypy/goal/targetpypystandalone_translate_only.py	2018-05-21 22:03:38.000000000 +0200
@@ -348,7 +348,10 @@
         pypy.module.sys.Module.interpleveldefs['pypy_translation_info'] = wrapstr
         if config.objspace.usemodules._cffi_backend:
             self.hack_for_cffi_modules(driver)
-
+        custom_goal = os.environ.get('PYPY_CUSTOM_GOAL')
+        if custom_goal:
+            driver.default_goal = custom_goal
+            driver.extra_goals = []
         return self.get_entry_point(config)
 
     def hack_for_cffi_modules(self, driver):
